src/main/java/bio/terra/model/StudyJsonConversion.java

```diff
@@ -0,0 +1,208 @@
+package bio.terra.model;
+
+import bio.terra.metadata.AssetColumn;
+import bio.terra.metadata.AssetRelationship;
+import bio.terra.metadata.AssetSpecification;
+import bio.terra.metadata.AssetTable;
+import bio.terra.metadata.Study;
+import bio.terra.metadata.StudyRelationship;
+import bio.terra.metadata.StudyTable;
+import bio.terra.metadata.StudyTableColumn;
+import bio.terra.model.RelationshipTermModel.CardinalityEnum;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public final class StudyJsonConversion {
+
+    // only allow use of static methods
+    private StudyJsonConversion() {}
+
+    public static Study studyRequestToStudy(StudyRequestModel studyRequest) {
```

Comment:
What are the preconditions? That is, what are you assuming about the validity of the incoming request? It seems that you are assuming it is well-formed; that the validation code has run. Be worth documenting that here.